### ISSUE TITLE
create Unison plugin

### ISSUE THREAD

**Issue by endphp2** [andrewreeman/SpectralSuite#1]
normally this effect is seen on the base instruments, rarely is an alone plugin.
Parameter:
voices 1-16
detune
stereo spread
phase differences
mod: Hz - octave
dry/wet

**Comment by andrewreeman** [comment]
Yes this is a good idea. I think this is usually accomplished as a time domain effect using delays. It would be interesting to do this as a frequency domain effect instead.

**Comment by endphp2** [comment]
multi pitch effect
